docs/docs/testcerts.md

```diff
@@ -0,0 +1,46 @@
+# Test Certificates
+
+To be able to start the SaltyRTC server, you need to specify a TLS key and
+certificate. In production you will want to use a certificate signed by a
+trusted CA, but for testing purposes, the easiest way is to create a
+self-signed certificate.
+
+## Generating a Test Certificate
+
+Use the following command to create such a certificate, valid for `localhost`
+during the next 5 years:
+
+    $ openssl req \
+           -newkey rsa:1024 \
+           -x509 \
+           -nodes \
+           -keyout saltyrtc.key \
+           -new \
+           -out saltyrtc.crt \
+           -subj /CN=localhost \
+           -reqexts SAN \
+           -extensions SAN \
+           -config <(cat /etc/ssl/openssl.cnf \
+             <(printf '[SAN]\nsubjectAltName=DNS:localhost')) \
+           -sha256 \
+           -days 1825
```

Comment:
Capping to 90 days should be sufficient. One doesn't want a test cert to be valid for a long period of time if it's meant for short-term testing. In case one forgets to delete it from Chrome, it can't do any harm (for long).

Reply:
To be honest it's a huge PITA having to generate a new test-cert and having to import it both into Firefox and Chrome every time I want to run SaltyRTC tests (which is often more than 90 days apart).

Reply:
Yes but you are an intelligent being that can adjust this parameter. :wink: